include/nebula/sclient/StorageClient.h

```diff
@@ -81,7 +81,6 @@ class StorageClient {
                                 std::string filter = "",
                                 bool onlyLatestVersion = false,
                                 bool enableReadFromFollower = true,
-                                bool needAuth = false,
                                 const std::string& username = "",
```

Comment:
prefer not to expose the username and password parameter in interface scan， users only need to config them once for StorageClient.